db/schema.rb

```diff
@@ -68,6 +55,19 @@
     t.index ["submitted_at"], name: "index_claims_on_submitted_at"
   end
 
+  create_table "decisions", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.integer "result"
+    t.string "checked_by"
+    t.uuid "claim_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.text "notes"
+    t.uuid "made_by_id"
```

Comment:
`created_by` would be consistent with the naming we use on `PayrollRun`. I think we should be consistent, given that the meaning of "created" and "made" is pretty much the same.

Reply:
I've updated our to be consistent with `PayrollRun`.

Part of me wonders whether we should use "Created by" on the UI or not, it might not be immediately understandable to end-users. However it's just an assumption so I've gone with the same as the payroll run UI for the time being:

<img width="674" alt="Screenshot 2020-02-11 at 12 12 37" src="https://user-images.githubusercontent.com/2804149/74236073-4c919180-4cc8-11ea-9c22-44d7d56a2cc2.png">

Reply:
I would say that the previous text ("Checked by") probably made more sense; do you think we need to change it?

Reply:
Because we're implementing this new concept of multiple checks that can be carried out by various different users I think we'd want to avoid "Checked by" for the decision. I think what we have now is ok, or perhaps "Decided by" but we can always iterate it later.

Reply:
Ah, good point. Cool, looks good to me then